tests/imodels-clients-tests/src/unit/management/AxiosRestClient.test.ts

```diff
@@ -0,0 +1,46 @@
+import { AxiosHeadersAdapterFactory, AxiosRestClient, IModelsErrorParser } from "@itwin/imodels-client-management/lib/base/internal";
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+import { expect } from "chai";
+
+import { ContentType, HttpRequestWithJsonBodyParams } from "@itwin/imodels-client-management";
+
+describe("[Management] AxiosRestClient", () => {
```

Comment:
`getAll` should also have a couple of tests

Reply:
added